girder_worker/plugins/docker/executor.py

```diff
@@ -67,9 +67,10 @@ def _expand_args(args, inputs, taskInputs, tmpDir):
             else:
                 val = ''
             arg = arg.replace('$flag{%s}' % inputId, val)
+            if not arg:
+                continue
```

Comment:
I was trying to preserve the behavior you had of excluding arguments in this replace case, but messed up.